perfkitbenchmarker/linux_benchmarks/cloudsuite_graph_analytics_benchmark.py

```diff
@@ -0,0 +1,160 @@
+# Copyright 2015 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs the graph analytics benchmark of Cloudsuite.
+
+More info: http://cloudsuite.ch/graphanalytics/
+"""
+
+import re
+
+from perfkitbenchmarker import configs
+from perfkitbenchmarker import errors
+from perfkitbenchmarker import flags
+from perfkitbenchmarker import sample
+from perfkitbenchmarker import vm_util
+from perfkitbenchmarker.linux_packages import docker
+
+flags.DEFINE_string('worker_mem',
+                    '2',
+                    'Ammount of memory for the worker, in gigabytes,'
+                    'use 0 to ignore this option')
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_NAME = 'cloudsuite_graph_analytics'
+BENCHMARK_CONFIG = """
+cloudsuite_graph_analytics:
+  description: >
+    Run Cloudsuite graph analytics benchmark. Specify the number of worker
+    VMs with --num_vms.
+  vm_groups:
+    master:
+      vm_spec: *default_single_core
+    workers:
+      vm_spec: *default_single_core
+      vm_count: 1
```

Comment:
It seems weird to me that the workers group, which has a configurable number of VMs, lists `vm_count: 1` here, while the master group, which should only have one VM, does not. I think this should be switched. The vm_count for the workers group is already overridden by GetConfig.